docs/migrations/version-3-to-4.md

```diff
@@ -0,0 +1,32 @@
+# Migration from v3 to v4
+This document contain all the breaking changes and migrations guidelines for adapting your code to the new version.
+
+## Property Naming changed for certain Edge Cases
+
+Object properties were formatted wrong when they contained a number followed by an underscore and a letter.
+This has been fixed in this version, which might means properties might be renamed. If you encounter errors, check for properties that were renamed.
```

Comment:
```suggestion
Naming such as object properties using camel case formatting had an edge case where if they contained a number followed by an underscore and a letter it would be incorrectly formatted. This has been fixed in this version, which might mean properties, model names, etc that use camel case might be renamed. 
```